Tactical/Overhead.cpp

```diff
@@ -10298,7 +10231,10 @@ void    DoneFadeOutDueToDeath( )
 void EndBattleWithUnconsciousGuysCallback( UINT8 bExitValue )
 {
     // Enter mapscreen.....
-    if(!is_client)CheckAndHandleUnloadingOfCurrentWorld();
+    if (!is_client)
+    {
+        CheckAndHandleUnloadingOfCurrentWorld();
+    }

```

Comment:
In the future, avoid mixing whitespace/formatting/etc changes with code changes. I know its tempting and I do it myself, but when reviewing it's balls